dandalandone/POC-EyCommerce
# [Frontend]: Rating Modal

## Ticket/s

- [Rating Modal Component](https://trello.com/c/0lbU4v8L/148-rating-component)

## Changes

- Rating modal
- Created reusable single star logo for rating

## Screenshot

- 
![image](https://github.com/TreshAnn/POC-EyCommerce/assets/132859942/e7f646e4-61cd-406f-ac34-ea31718812c9)
![image](https://github.com/TreshAnn/POC-EyCommerce/assets/132859942/1b68baf5-c402-4cdd-883e-0e800ea759ed)


## How to test?

- checkout to branch
- navigate to SampleView "/"
- Click "rate me" button

*see notes in SampleView
